packages/components/src/responsive-wrapper/index.js

```diff
@@ -8,19 +8,34 @@ import classnames from 'classnames';
  */
 import { cloneElement, Children } from '@wordpress/element';
 
-function ResponsiveWrapper( { naturalWidth, naturalHeight, children } ) {
+function ResponsiveWrapper( {
+	naturalWidth,
+	naturalHeight,
+	children,
+	isInline = false,
+} ) {
 	if ( Children.count( children ) !== 1 ) {
 		return null;
 	}
 	const imageStyle = {
 		paddingBottom: ( naturalHeight / naturalWidth * 100 ) + '%',
 	};
+	const wrapperClass = 'components-responsive-wrapper';
+	const childrenProcessed = cloneElement( children, {
+		className: classnames( 'components-responsive-wrapper__content', children.props.className ),
+	} );
+	if ( isInline ) {
```

Comment:
if it's the exact same code, can't we just use `const TagName = isInline ? "span" : "tag"` to avoid duplicating it?

Reply:
Yes it seems to reduce duplicate code, I update following your suggestion 👍